packages/client/test/stream.test.ts

```diff
@@ -66,6 +79,64 @@ describe("Stream", async () => {
     }
   });
 
+  test("Multiple Stream Creation", async () => {
+    const testCreateItem = {
+      updateItem: {
+        eventId: Buffer.from([1, 2, 3, 4]),
+        price: "1000",
+        metadata: Buffer.from([1, 2, 3, 4]),
+      },
+    };
+
+    const signedMessage = await signMessage(testCreateItem);
+    const events = [];
+
+    for (let index = 0; index < 50; index++) {
+      events.push(signedMessage);
+    }
+    const pushEvent = {
+      requestId: Uint8Array.from([1, 2, 3, 4]),
+      events,
+    };
+    const client = new MockClient();
+    const stream = new ReadableEventStream(client);
+    stream.enqueue(pushEvent);
+    let count = 0;
+    for await (const evt of stream.stream) {
+      count++;
+      if (count === pushEvent.events.length) break;
+    }
```

Comment:
add expect(count).toEqual(...

Reply:
that doesn't break/cancel the draining of the stream though